packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -173,7 +174,7 @@ module.exports = function createBaseConfig(env) {
 
 		module: {
 			rules: [
-				{
+				(info) => ({
```

Comment:
The tests seem to be failing because this is a function

Reply:
Is this only supported in newer Webpack?

Reply:
No, isn't supported in newer Webpack either. Rules have to be objects.